Doc/library/asyncio-stream.rst

```diff
@@ -106,11 +116,41 @@ and work with streams:
 
    .. deprecated-removed:: 3.8 3.10
 
-      `start_server()` is deprecated if favor of `StreamServer()`
+      `start_server()` is deprecated if favor of :class:`StreamServer`
+
+.. coroutinefunction:: connect_read_pipe(pipe, *, limit=2**16)
+
+   Takes a :term:`file-like object <file object>` to return a
+   :class:`Stream` object of the mode ``READ`` that has similar API of
+   :class:`StreamReader`. It can also be used as an async context manager.
+
+   .. versionadded:: 3.8
+
+.. coroutinefunction:: connect_write_pipe(pipe, *, limit=2**16)
+
+   Takes a a :term:`file-like object <file object>` to return a
+   :class:`Stream` object of the mode ``WRITE`` that has similar API of
+   :class:`StreamWriter`. It can also be used as an async context manager.
 
+   .. versionadded:: 3.8
 
 .. rubric:: Unix Sockets
 
+.. function:: connect_unix(path=None, *, limit=2**16, ssl=None, \
+                           sock=None, server_hostname=None, \
+                           ssl_handshake_timeout=None)
+
+   Establish a Unix socket connection and return an awaitable :class:`Stream`
+   object of the mode ``READWRITE`` that can be used as a reader and a writer.
+   The function can also be used as an async context manager where it's
+   automatically awaited.
+
+   Similar to :func:`connect` but operates on Unix sockets.
+
```

Comment:
Please describe the rest arguments: limit should be explained directly, all others can be mentioned by a link to `loop.create_unix_connection()`

Reply:
Added doc for limit like above functions and added a note that rest of arguments are passed to `loop.create_unix_connection()`. Thanks